CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 Date format: DD/MM/YYYY
 
+## [unreleased]
+
+- `AutoSuggestBox` dynamic type support ([#441](https://github.com/bdlukaa/fluent_ui/issues/441))
```

Comment:
this is a breaking change.

this is just a draft. can be done better
```suggestion
- **BREAKING** `AutoSuggestBox` dynamic type support ([#441](https://github.com/bdlukaa/fluent_ui/issues/441))
  Before:
  ```dart
AutoSuggestBox(
  items: cats.map((cat) {
    return AutoSuggestBoxItem(
      value: cat,
      onFocusChange: (focused) {
        if (focused) debugPrint('Focused $cat');
      }
    );
  }).toList(),
  onSelected: (item) {
    setState(() => selected = item);
  },
),
  ```Now:
  ```dart
AutoSuggestBox<String>(
  items: cats.map((cat) {
    return AutoSuggestBoxItem<String>(
      value: cat,
      label: cat,
      onFocusChange: (focused) {
        if (focused) debugPrint('Focused \$cat');
      }
    );
  }).toList(),
  onSelected: (item) {
    setState(() => selected = item);
  },
),
  ```

```

Reply:
Added.